src/rpcwallet.cpp

```diff
@@ -94,7 +94,7 @@ void WalletTxToJSON(const CWalletTx& wtx, UniValue& entry)
 string AccountFromValue(const UniValue& value)
 {
     string strAccount = value.get_str();
-    if (strAccount == "*")
+    if (strAccount.empty())
```

Comment:
This backfired. Now empty string labels cannot be selected anymore. I think for now it's best to revert to the previous state with `""` selecting all coins and `"*"` not allowed and at a later point port bitcoin's `GetLegacyBalance`. Sorry for my opinion slalom.

Reply:
I also tested it with the previous state of using `""` to select all, and it worked fine.

Reply:
haha yea i realised that when u said that whoops i can opt out of AccountFromValue in that function lol